common/buildcraft/BuildCraftCore.java

```diff
@@ -364,6 +364,9 @@ public void loadConfiguration(FMLPreInitializationEvent evt) {
 		FMLCommonHandler.instance().bus().register(this);
 		MinecraftForge.EVENT_BUS.register(this);
 		MinecraftForge.EVENT_BUS.register(new BlockHighlightHandler());
+
+		OreDictionary.registerOre("chestWood", Blocks.chest);
+		OreDictionary.registerOre("craftingTableWood", Blocks.crafting_table);
```

Comment:
maybe check if it isn't registered yet? not sure what the effect of double entries would be